packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -103,15 +130,15 @@ export const getExplorerUrl = (chainId: ChainId) => {
 }
 
 export const getBlockTime = (chainId: ChainId) => {
-  const network = l1Networks[chainId]
+  const network = parentChains[chainId]
   if (!network) {
     throw new Error(`Couldn't get block time. Unexpected chain ID: ${chainId}`)
   }
-  return network.blockTime
+  return (network as L1Network).blockTime ?? 12
```

Comment:
`blockTime` doesn't exist on `L2Network`. `network` is of type `ParentChain` which can be an Arbitrum chain.